src/grocker/resources/docker/root-image/provision.sh.j2

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+set -xe
+
+debian_setup () {
+    # Adapt APT config for Docker
+    echo 'force-unsafe-io' > /etc/dpkg/dpkg.cfg.d/docker-apt-speedup
+    cat <<EOF > /etc/apt/apt.conf.d/docker-clean
+    DPkg::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
+    APT::Update::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
+
+    Dir::Cache::pkgcache ""; Dir::Cache::srcpkgcache "";
+
+    Acquire::Languages "none";
+
+    APT::Install-Recommends=false;
+    APT::Install-Suggests=false;
+EOF
+
+{% if repositories %}
+    apt update
+    apt install -qy apt-transport-https
+    apt-get clean
+
+{% for name, desc in repositories.items() %}
+    apt-key add - <<EOF
+{{ desc['key'].strip() }}
+EOF
+    echo "{{desc['uri']}}" > /etc/apt/sources.list.d/{{name}}.list
+{% endfor %}
+{% endif %}
+
+    # Install System Packages
+    export DEBIAN_FRONTEND=noninteractive
+    apt update
+    apt upgrade -qy
+    apt install -qy ${SYSTEM_DEPENDENCIES}
+    apt-get clean
+
+    # Create grocker user
+    adduser --shell /bin/bash --disabled-password --gecos ",,,," grocker
+
+    # Configure system
+    echo "LANG=C.UTF-8" > /etc/default/locale
+}
+
+alpine_setup() {
+    echo "TODO: Deal with repos"
+
+    apk update
+    apk upgrade
+    apk add ${SYSTEM_DEPENDENCIES:=}
+    rm -rf /var/cache/apk/*
+
+    # Create grocker user
+    adduser -s /bin/sh -D -g ",,,," grocker
+
+    # No need to configure system's locale
```

Comment:
Maybe it's easier to put the LANG envvar in the Dockerfile, it will work with all POSIX systems